mysql-persistence/src/main/java/com/netflix/conductor/dao/mysql/MySQLMetadataDAO.java

```diff
@@ -263,6 +263,7 @@ public void removeEventHandlerStatus(String name) {
     private void validate(TaskDef taskDef) {
         Preconditions.checkNotNull(taskDef, "TaskDef object cannot be null");
         Preconditions.checkNotNull(taskDef.getName(), "TaskDef name cannot be null");
+        Preconditions.checkNotNull(taskDef.getResponseTimeoutSeconds(), "ResponseTimeoutSeconds cannot be null");
```

Comment:
Also check responseTimeoutSeconds must be greater than 0 (i.e. it can't be 0 or negative)